### DISCUSSION TITLE
No "Import from server" menu item ...

### DISCUSSION THREAD

**Discussion by dream7180** [LycheeOrg/Lychee#2767] [Q&A] [answered]
Lychee v6.1.2 installed in archlinux, caddy web server.

1) I have log in as admin. But "import from server" is not available in the upload menu. Is there anything missing?
![图片](https://github.com/user-attachments/assets/560d10a5-cecf-4096-add1-c8deb7d4ccaa)

2) I have install imagemagick and php-imagick in the system, But still there is below warning. What can I do further?
![图片](https://github.com/user-attachments/assets/97235afe-2af7-4420-b3d6-9f4440c95a60)

**Comment by ildyria** [comment, marked as answer]
Import from server has been dropped since v5 already.

You can use the `php artisan lychee:sync` command to do the work.
See more here: https://github.com/LycheeOrg/Lychee/blob/master/app/Console/Commands/Sync.php#L25
Or alternatively you can temporary revert the front-end to version 4 by changing your `.env`  with `VUEJS_ENABLED=false`. Note that this will be using the old API.

With regard to imagick, you should check your phpinfo to see if it is really enabled : https://lychee.test/phpinfo

**Reply by dream7180** [reply]
Thank you, ildyria.
